hikari/interactions/command_interactions.py

```diff
@@ -24,16 +24,12 @@
 from __future__ import annotations
 
 __all__: typing.List[str] = [
-    "Command",
-    "CommandChoice",
     "CommandInteractionOption",
     "CommandInteraction",
-    "CommandOption",
     "COMMAND_RESPONSE_TYPES",
     "CommandResponseTypesT",
     "InteractionChannel",
     "ResolvedOptionData",
-    "OptionType",
 ]
```

Comment:
Why not move everything to commands.py?

Reply:
Like detailed before ths just more strictly follows the current pattern where dispatch classes are kept separate from general entity classes